### PR TITLE
added funny cores to cargo bounties

### PR DESCRIPTION
**Pull request by MarkusLarsson421**
## About The Pull Request
Adds all the anomalies (except insanity) to be a Cargo bounty.
They only require one (despite the testing evidence) and reward the following (despite the testing evidence):

bleed, 25 000 
bluespace, 45 000 
delimber, 30 000 
flux, 20 000 
gravity, 20 000
hallucination, 15 000 
pyro, 25 000 
vortex, 50 000

It's value has been balanced on my personal opinion of its rarity and danger.

The cores now also has a regular sell value at Cargo.

bleed, 5 000
bluespace, 10 000
delimber, 3 000
vortex, 15 000

These anomalies are farmable by the SM and therefore has a lower value than the rest.

gravity, 500
hallucination, 250
flux, 500
pyro, 750

## Why It's Good For The Game
Makes Taylor happy and makes Cargo WANT to get those pesky anomalies. Could encourage Engineers to make a spicy SM to make it create some anomalies if there is a bounty for it.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>
The money and amount has been changed, look at About The Pull Request for more detailed information.

image

image

</details>

## Changelog
:cl:
add: Added anomaly cores to the Cargo bounty.
add: Added cores to be sellable, including ones made from the SM.
balance: Added another method for Cargo to get fat stacks of cash.
balance: Gave anomaly cores a non-bounty sell value, even ones made by the SM.
/:cl:

### CODE REVIEW

**Review by Rukofamicom — APPROVED** [review]

**Comment by MarkusLarsson421** [comment]
Was told that the rewards were too low, so I'll increase it when I have the time.

**Comment by Rukofamicom** [comment]
Were you planning to include a normal sale price on them here too?

**Comment by MarkusLarsson421** [comment]
> Were you planning to include a normal sale price on them here too?

I could do that, sure.

**Review by Rukofamicom — APPROVED** [review]

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
These prices are way too high. They should be 6000 at most.

**Comment by Rukofamicom** [comment]
Which prices, the bounty prices or the normal prices? Most bounty prices are quite high - simple crafted items from lavaland range up to 36,000 credits.

**Comment by MarkusLarsson421** [comment]
Normal sell prices I think he refers to.

Considering their rarity, I don't think the prices are that bad. Especially not the Vortex and Bluespace ones which are REALLY rare and which Science would actually be interested in because of their unique properties.

I won't change anything immediately but give this some thought for a day or two and reach back then.

**Review by PowerfulBacon — APPROVED** [review]
Didn't realise those were bounty prices. I don't think vortex anomalies actually do anything useful for science though

**Comment by MarkusLarsson421** [comment]
> I don't think vortex anomalies actually do anything useful for science though

It can be used with Reactive Armour and the Phazon. I've heard it gives infinite power to a Phazon, but haven't tested this out. Either way, it is rare as hell.

File: code/modules/cargo/exports/core.dm [hunk 1]
```diff
@@ -0,0 +1,36 @@
+/datum/export/core
+
+/datum/export/core/bleed
+	cost = 5000
+	unit_name = "bleed"
+	export_types = list(/obj/item/assembly/signaler/anomaly/)
```
Comment (Tsar-Salat) [inline code/modules/cargo/exports/core.dm:6]: why the hell

Comment (MarkusLarsson421) [reply code/modules/cargo/exports/core.dm:6]: Care to elaborate or are you just necroposting?

Comment (Tsar-Salat) [reply code/modules/cargo/exports/core.dm:6]: why are they all the parent type

Comment (MarkusLarsson421) [reply code/modules/cargo/exports/core.dm:6]: No idea, why not?

**Pull request merged**